Package.resolved

```diff
@@ -1,15 +1,6 @@
 {
   "object": {
     "pins": [
-      {
```

Comment:
Wonder if we should remove `Package.resolved` for now to avoid dev churn.

Reply:
agreed, will do